### ISSUE TITLE
Scrolling a stack with non-square PixelSpacing resets the camera

### ISSUE THREAD

**Issue by RoyWiggins** [cornerstonejs/cornerstone3D-beta#548]
If I have a StackViewport with a stack of dicoms that have a non-square `PixelSpacing`, switching between images causes the camera to reset to its initial position. This is awkward and makes it impossible to zoom in on a cine stack and scroll through time. 

If the `PixelSpacing` is not present or is `1.0\1.0`, it behaves how I expect and the camera doesn't reset.

**Comment by sedghi** [comment]
I agree that is awkward, is the data anonymized and whether can you share it?

**Comment by RoyWiggins** [comment]
Here's a synthetic dataset that exhibits the problem. 

[test_juliaset.zip](https://github.com/cornerstonejs/cornerstone3D-beta/files/11253998/test_juliaset.zip)

**Comment by sedghi** [comment]
can you try now?

**Comment by RoyWiggins** [comment]
So, it turns out that in my actual project, we are further behind the main branch that I remembered. I have been struggling a bit to catch up without producing errors deep inside the renderer... I will need to spend some more time with that. 

In the meantime, I tried manually backporting the relevant changes. It seems to work! 

The only issue is- and I think this may be due to my incomplete backport-  `isEqual(imagePlaneModule.rowCosines, <Point3>rowCosines)` is returning `False` due to rowCosines actually being a Float64Array. Changing the relevant lines to use `Array.from()` seemed to fix that:

```
      isEqual(imagePlaneModule.rowCosines, Array.from(rowCosines)) &&
      isEqual(imagePlaneModule.columnCosines, Array.from(columnCosines))
```

This may not actually be relevant to the fix though, but I thought I'd mention it just in case.
